util/cpb/main.go

```diff
@@ -128,3 +161,107 @@ func getMetadata() (m *metadata, err error) {
 
 	return
 }
+
+// copyIndex searches for the index in an image and then copies it to dest
+// it will create intermediate directories for dest if they do not exist
+func copyIndex(dest string) (err error) {
```

Comment:
nit: unnecessary parentheses around the return error

Reply:
they are necessary since we're using a named return parameter